test/e2e/kubernetes/scripts/host-os-dns-validate.sh

```diff
@@ -13,11 +13,11 @@ HOSTS="${NODE_HOSTNAMES} www.bing.com google.com"
 for i in $(seq 1 $retries); do
   for host in $HOSTS; do
     set -x
-    [ -s $(dig +short +search +answer ${host}) ]
-      if [ $? -eq 0  ]; then
-        success="no"
-        break
-      fi
+    addrs=$(dig +short +search +answer ${host})
```

Comment:
This one was confusing: `dig` doesn't return 0 on a failure to find records, so I assume `-s` was attempting to see if the command returned a non-empty file...but that syntax didn't work for me in macOS `bash` or on Ubuntu.

I think assigning to `addrs` and testing with `-z` for zero-length is more readable and does the job, but maybe I misunderstood the original command.